### PR TITLE
Move badge

### PR DESCRIPTION
**Pull request by tomclose**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "ordered-imports": false
   },
   "rules": {
+    "no-console": [true, "warning"],
```
Comment (kerzhner) [inline tslint.json]: I probably agree with this.

File: src/components/OpenGameCard.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export class OpenGameEntry extends React.PureComponent<Props, State> {
 
     const stake = openGame.stake;
     // TODO: should not hardcode the number of rounds
-    const buyin = stake.mul(new BN(5));
+    const buyin = new BN(5000000000000000);
```
Comment (kerzhner) [inline src/components/OpenGameCard.tsx]: @tomclose Does the buy-in depend on the wager per round?

File: src/components/MoveBadge.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  hoverOff() {
+    this.setState({ hover: false });
+  }
+
+  // todo: get hover to change the appearance
+  render() {
+    const { move, action } = this.props;
+    switch (move) {
+      case Move.Rock:
+        return <img className={`move-badge`} src={ROCK_BADGE} onClick={action} onMouseEnter={this.hoverOn} onMouseLeave={this.hoverOff} />;
```
Comment (kerzhner) [inline src/components/MoveBadge.tsx]:
```suggestion
        return <img className='move-badge' src={ROCK_BADGE} onClick={action} onMouseEnter={this.hoverOn} onMouseLeave={this.hoverOff} />;
```

File: src/components/OpenGameCard.tsx (hunk 2 above)
Comment (tomclose) [reply src/components/OpenGameCard.tsx]: This is a temporary fix to work around `BN` problems. Will be dealing with this next.

**Pull request merged**